tests/tests/iframe_script.spec.ts

```diff
@@ -16,17 +16,20 @@ test.describe('Iframe API', () => {
   });
 
   test('can add a custom menu by scripting API', async ({
-                                                                 page,
-                                                               }) => {
+    page
+  }) => {
     await page.goto(
         'http://play.workadventure.localhost/_/global/maps.workadventure.localhost/tests/Metadata/customMenu.json'
     );
 
     await login(page);
 
-    await page.click('#menuIcon img:first-child');
-
-    await page.click('button:has-text("custom iframe menu")');
+    // wait some time for the scripting menu to be loaded
+    // eslint-disable-next-line playwright/no-wait-for-timeout
+    await page.waitForTimeout(2_000);
```

Comment:
Ach! Verbotten!

I'm doing my best to keep timeouts out of the tests. They are too slow already. We need to replace that with a proper "expect".